### DISCUSSION TITLE
你都用哪些社交軟體呢?

### DISCUSSION THREAD

**Discussion by Superlan8** [1101-GS4719-C/Python-Community-GS4719-C#1315] [General]
當今社交媒體的普及率已經達到了空前的高度，越來越多的人使用社交軟體來聯繫家人朋友，分享日常生活，以及發現新的社交圈子。然而，社交媒體的使用率不僅僅是一種簡單的趨勢，它也反映了當代社會的一些核心價值觀和行為方式。

我在看一些街訪，發現現在大部份年輕人傾向使用Instagram，但看數據發現，Instagram才以65.3%的覆蓋率囊括第三名而已，遠輸第二名Facebook的85.3%。我認為其最大原因為下，Facebook畢竟是最早進入台灣市場的社交軟體，擁有較穩固的用戶基礎，並且功能相較於Instagram多一些，比較符合大部分台灣人的使用習慣。

而第一名毫無疑問的是Line的90.7%，原本就夠穩定的用戶基礎加上最近火熱的線上支付Line pay，以及能針對業者設置的Line bot，以上種種原因皆是我認為Line可以在台灣獨居龍頭的關鍵。

想問大家你最常使用哪些社交軟體，而你都用來幹嘛，你認為它相較其他軟體的優勢是甚麼?

**Comment by llh26** [comment]
想針對聊天室功能去做討論！個人覺得LINE很適合保留「照片」，畫質相較於其他軟體好很多，也可以建立相簿，真的非常方便～貼圖種類和風格也是它吸引人的地方，其他軟體很難做到這種程度。而Facebook的Messenger很適合傳檔案，因為不會有過期的問題（LINE只有7天的存取期限），所以自己在選擇上，會衡量自己和另一方比較需要哪一方面的交流，處理事情也比較方便！

**Comment by quan0715** [comment]
我覺得 Line 跟 Messenger 都很好但比較僅限於聊天軟體，其實聽說有些理工人會吧 Github 當作社交軟體在經營，還蠻酷的！！

**Comment by Jamesmoneya** [comment]
其實line相比其他聊天的軟體，功能實在為人詬病，現在就是因為大部分的人都在用，如果你不用根本不會有人為了你去下載其他的應用程式，line公司利用他的獨佔的優勢，進步的非常緩慢，連轉個資料都只能薄存一下下，很快就過期，還不能輩分聊天紀錄到不同系統的手機上

**Comment by TW-Maverick** [comment]
我還是比較習慣使用FB👍，因為上面有比較多的科學新知跟知識，Ex.NASA Artemis、NASA JPL
還有社團可以互相討論，這是instagram所沒有的，
而且FB還可以顯示誰生日是什麼時候🤣

**Comment by Xunhaoz** [comment]
唯一指定 VSinder 可以寫程式也可以交友，何樂而不為？
https://technews.tw/2020/12/24/vsinder-vs-code-tinder/

**Comment by alicia0725** [comment]
以聯絡跟討論公事為目的基本上還是用Line居多，對我而言，FB的用處更像是先用來聯絡不太認識的人，以及社團的功能。而instagram則更像是很單純的娛樂軟體、用來分享生活居多。因為fb是我從小就使用的軟體，感覺更容易看到一些以前的隱私

**Comment by lyien1021** [comment]
我自己是比較喜歡Instagram的介面，Facebook對我來說有點太雜亂了，不過因為學校很多社團都是使用Facebook所以不得不用...最近instagram也有推出類似社團的功能，不知道以後有沒有可能取代Facebook社團

**Comment by a3240281370** [comment]
根據一些研究，FB的使用者年齡分佈相對偏高，主要集中在30歲以上的年齡段。IG在年輕人中非常受歡迎，尤其是年輕成年人和青少年。許多研究表明，IG的主要使用者群體是18至34歲的年輕人，尤其在20多歲的年齡段。LINE是一個主要在亞洲地區流行的即時通訊應用程序。LINE的使用者年齡分佈相對較廣泛，包括年輕人和成年人，但在一些報告中指出，它在30歲以下的年齡段中更受歡迎。

**Comment by juliaouo** [comment]
我非常喜歡FB的社團功能，可以認識許多不同的人，另外發文也不用帶圖我覺得很棒。聊天的話我常用 Messenger 跟 Telegram，Messenger是因為可以彈出對話頭像，避免忘記回，TG 則是因為可以自己製作貼圖，看到喜歡的也都可以拿來用，另外傳圖檔什麼的都可以保留，之後要查找也很方便，並且自定義功能非常多，我認為非常好用！

**Comment by 1104030360** [comment]
我最近都是使用ig，因為我追蹤的人事物都是我想看的，不會跳出太多廣告！

**Comment by JessyCheng2003** [comment]
我最常用的社交軟體是Instagram 我覺得他很適合用來分享生活和照片 其中我最喜歡的功能是「限時動態」可以用來po日常發生的事或是美食 再加上它24小時過後就會自動刪除 不會一直保留著 讓大家可以隨心所欲的分享～

**Comment by harrycoconut** [comment]
我最常使用的是instagram及line，在instagram中能看到大家的限時動態，讓自己不會與世界脫節，而line的話主要是跟朋友和重要的廠商聯絡，真的非常方便😌

**Comment by Tzuhsuannnnn** [comment]
我覺得Instagram,fb,line都是我不能捨棄的3個社交軟體，每個都有自己的優點，像line的貼圖跟聊天方便性就很好，但備份功能就做得不太好，導致很多人在轉移手機時，常有資料或帳號丟失的困擾。而fb則是使用者年齡層很廣。Instagram的限時動態跟reels也都是分享自己生活很好用的軟體。

**Comment by pigeon9975** [comment]
討論或線上聊天我LINE、MSG、IG都會使用，而營隊或小組報告我傾向用MSG，因為MSG可以看誰已讀的功能非常方便，做報告時有人擺爛就可以馬上抓出來！阿IG跟FB主要是拿來體驗別人生活、看梗圖、迷因用的。
